### PR TITLE
Search product

### PR DESCRIPTION
**Pull request by hoangnbvn**
## Related Tickets:
- [#68015](https://edu-redmine.sun-asterisk.vn/issues/68015)

## WHAT this PR does?
- Search product by name, brand,...

## Notes Impacted Areas:
- app/Http/Controllers/ProductController.php
- resources/views/
- routes/web.php
## Attachments:
image

### CODE REVIEW

**Comment by BuiMinhDung711** [comment]
ok

**Comment by hoangnbvn** [comment]
ready

File: resources/css/app.css [hunk 1]
```diff
@@ -890,3 +890,12 @@ span.text-red {
...
 }
+.filter-form {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.filter-group {
+    margin-right: 10px;
+}
```
Comment (ghost) [inline resources/css/app.css]: eof

File: resources/views/product/search.blade.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        <div class="products-container">
+            @foreach ($products as $product)
+                <x-product id="{{ $product->id }}" 
+                    media-link="{{ $product->media_link }}" 
+                    name="{{ $product->name }}" 
+                    price="{{ $product->price }}" />
+            @endforeach
+        </div>
+    @else
+        <p>No results found.</p>
```
Comment (ghost) [inline resources/views/product/search.blade.php]: localization

**Comment by hoangnbvn** [comment]
ready

**Pull request merged**